build.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+const fs = require('fs');
+const https = require('follow-redirects').https;
```

Comment:
Are you sure this is actually needed? I don't think the URL redirects to anything

Reply:
I will make a ss so you can see what happens without ```follow-redirects```

Reply:
Now I feel stupid because now it works without it but there was a redirect page when I tried to access it the first time.

Reply:
Ahah, it was probably because you were using the `http` URL.
If you load `http://uplift.shipit.staging.mozilla-releng.net/coverage/supported_extensions` it will redirect you to `https://uplift.shipit.staging.mozilla-releng.net/coverage/supported_extensions`